med_io/pipeline.py

```diff
@@ -89,14 +90,19 @@ def _map(*args):
 
             patchs_imgs, patchs_labels, index_list = get_patches_data(max_data_size, patch_size, images_data,
                                                                       labels_data,
-                                                                      patches_indices, slice_channel_img=input_slice,
+                                                                      patches_indices,
+                                                                      slice_channel_img=input_slice,
                                                                       slice_channel_label=output_slice,
                                                                       output_patch_size=config['model_output_size'],
+                                                                      random_shift_patch=config['random_shift_patch'],
                                                                       squeeze_channel=config['squeeze_channel'])
 
+
             # List regularize
             #index_list = index_list / (np.array(max_data_size) + 1e-16)
-            if config['feed_pos']:
+            if config['active_learning']:
```

Comment:
add to config check, default: False

Reply:
Added the required config check (util.convert_yaml_config() line 37)

Reply:
thanks